src/mod/irc.mod/chan.c

```diff
@@ -1216,25 +1241,11 @@ static int got352or4(struct chanset_t *chan, char *user, char *host,
   /* Update accountname in channel records, 0 means logged out */
   /* A 0 is not a change from "" */
   if (account) {
```

Comment:
This gets tricky.... this function is triggered for every WHOX, but the account field we're looking for depends on the WHOX format field being "c%%chnufat,222". If a Tcl script sends "%cuhnar,42" as the format field, Eggdrop does this:
```
[22:04:05] BeerBot!user@b0rg-C0E59E12.foo.net logged in to their account :realname
```
And .channel shows the account name as ":realname"

Reply:
we ignore the response in got354 if the query type isn't 222 so mismatch isn't possible unless scripts are malicious, the colon must be from gotaccount

Reply:
nuh-uh!